### PR TITLE
Helpful Development Plugins - React - Closes #231

### LINKED ISSUE #231: Helpful Development Plugins
**Is your feature request related to a problem? Please describe.**
A list of your most helpful web plugins

**Describe the solution you'd like**
A helpful list of safe plugins we've found to be helpful while developing.

Some common web plugins for React Development:

- [React Developer Tools](https://github.com/facebook/react/tree/master/packages/react-devtools-extensions)
- [Redux DevTools Extension](https://github.com/zalmoxisus/redux-devtools-extension)

If you know of any others feel free to add them!

### PR DESCRIPTION
**Pull request by caroham29**
## Changes
_List Changes Introduced by this PR_
1. Add to list of safe and helpful development plugins
2. Specific to React

## Purpose
Create a location to share helpful plugins used by our development team. Adding a couple of resources commonly used in React. 

## Approach
This change will give us a centralized location of helpful plugins related to development in React.

- [React Developer Tools](https://github.com/facebook/react/tree/master/packages/react-devtools-extensions)
- [Redux DevTools](https://github.com/zalmoxisus/redux-devtools-extension)

closes #231

### CODE REVIEW

File: best-practices/client-side/framework/react/README.md [hunk 1]
```diff
@@ -20,6 +20,11 @@ To get more familiar with the structure of a React/React Native based project, b
 -   [Jest](https://jestjs.io/) is a JavaScript library for creating, running, and structuring tests.
 -   [Enzyme](https://enzymejs.github.io/enzyme/) created by Airbnb is a JavaScript Testing utility for React.
 
+## Helpful React Development Web Plugins
```
Comment (caroham29) [inline best-practices/client-side/framework/react/README.md:23]: *Revised* Day 3 Checklist! Here's a couple of helpful React development web plugins. 👍🏾

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
@caroham29 The only thing preventing this PR being approved is the lack of a referenced issue. All PRs must have an issue attached before a PR is approved.

Basically, just add Closes #999 <-- issue number here
to the end of your current PR and you should be good.

I look forward to having more react based tools in our standards!

**Review by codysmith287 — APPROVED** [review]
Nice addition here! I've used both of these web plugins before and they've been super helpful for debugging!

**Comment by coreyshuman** [comment]
@caroham29 can you put the `closes #231` into the body of your PR post, instead of the title? GitHub has some automation smarts built in, so if you include that line in the body, it will automatically close the Issue once the PR is merged. It will also create a hyperlink reference between the PR and the Issue.

Check out this recent PR for an example: https://github.com/Shift3/standards-and-practices/pull/228

Here's the GitHub documentation on this feature.
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

**Comment by caroham29** [comment]
> @caroham29 can you put the `closes #231` into the body of your PR post, instead of the title? GitHub has some automation smarts built in, so if you include that line in the body, it will automatically close the Issue once the PR is merged. It will also create a hyperlink reference between the PR and the Issue.
> 
> Check out this recent PR for an example: #228
> 
> Here's the GitHub documentation on this feature.
> https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

@coreyshuman Done, and thanks for the documentation link.

**Review by michaelachrisco — APPROVED** [review]
Thanks for the edit. Approved!

**Pull request merged**